solutions/swb-reference/src/environment/sagemakerNotebook/sagemakerNotebook.cfn.yaml

```diff
@@ -70,6 +70,13 @@ Resources:
           FromPort: 443
           ToPort: 443
           CidrIp: !Ref AccessFromCIDRBlock
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W5
+            reason: 'At this point, this is by design. Customers should only be using this with test data.'
+          - id: W36
+            reason: 'There is clearly a description provided. Not sure why CFN nag is not picking it up from properties.'
```

Comment:
I'll look into this with you when you come back to triage. Might be worth filing an issue on cfn_nag